packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -2,15 +2,18 @@
 
 .root {
   @include focus-border();
-  flex-shrink: 0;
+  flex-shrink: 1;
   background: $ms-color-neutralLighter;
   height: 32px;
   line-height: 32px;
   cursor: default;
   user-select: none;
   display: flex;
   flex-wrap: nowrap;
+  min-width: 100px;
   max-width: 100%;
+  margin: 2px 2px;
```

Comment:
Yeah, I found those problems when doing the box height issues. Go ahead and back them out and I'll make sure they're resolved in my PR